[2024-02-21T06:37:21.338+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-21T06:37:10.072116+00:00 [queued]>
[2024-02-21T06:37:21.359+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-21T06:37:10.072116+00:00 [queued]>
[2024-02-21T06:37:21.360+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T06:37:21.403+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): download_from_s3> on 2024-02-21 06:37:10.072116+00:00
[2024-02-21T06:37:21.424+0000] {standard_task_runner.py:60} INFO - Started process 922 to run task
[2024-02-21T06:37:21.477+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '***-s3-mlflow', 'download_from_s3', 'manual__2024-02-21T06:37:10.072116+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/***-aws-mlflow.py', '--cfg-path', '/tmp/tmpg3rfi9fc']
[2024-02-21T06:37:21.510+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask download_from_s3
[2024-02-21T06:37:23.067+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-21T06:37:23.368+0000] {task_command.py:423} INFO - Running <TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-21T06:37:10.072116+00:00 [running]> on host 4d6165970785
[2024-02-21T06:37:23.673+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-s3-mlflow' AIRFLOW_CTX_TASK_ID='download_from_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-21T06:37:10.072116+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-21T06:37:10.072116+00:00'
[2024-02-21T06:37:23.676+0000] {s3.py:1375} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2024-02-21T06:37:23.677+0000] {s3.py:1381} INFO - Downloading source S3 file from Bucket airdemobkt with path wine-quality.csv
[2024-02-21T06:37:23.706+0000] {base_aws.py:581} WARNING - Unable to find AWS Connection ID 's3_conn', switching to empty.
[2024-02-21T06:37:23.708+0000] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2024-02-21T06:37:24.495+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-02-21T06:37:44.481+0000] {python.py:201} INFO - Done. Returned value was: /opt/***/data/***_tmp_ysc3ea4l
[2024-02-21T06:37:44.510+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=***-s3-mlflow, task_id=download_from_s3, execution_date=20240221T063710, start_date=20240221T063721, end_date=20240221T063744
[2024-02-21T06:37:44.579+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T06:37:44.618+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
